src/site/content/en/blog/storage-foundation/index.md

```diff
@@ -0,0 +1,372 @@
+| 5. Launch                                | Not started              |
+
+</div>
+
+{% Aside %} There is currently an
+[ongoing effort](https://docs.google.com/document/d/1g7ZCqZ5NdiU7oqyCpsc2iZ7rRAY1ZXO-9VoG4LfP7fM/edit?usp=sharing)
+to augment the
+[origin private file system](http://localhost:8080/file-system-access/#accessing-the-origin-private-file-system)
+of the [File System Access API](/file-system-access/) as to not introduce yet another entry point
+for a storage system. This article will be updated as we make progress on this. {% endAside %}
+
+## How to use the Storage Foundation API {: #use }
+
+### Enabling via about://flags
+
+To experiment with the Storage Foundation API locally, without an origin trial token, enable the
+`#experimental-web-platform-features` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;90, the Storage Foundation API will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;95 (November 10, 2021).
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Feature detection
+
+To check if the Storage Foundation API is supported, use:
+
+```js
+if ('storageFoundation' in window) {
+  // The Storage Foundation API is supported.
+}
+```
+
+### Complete examples
+
+To make the concepts introduced above clearer, here are two complete examples that walk you through
+the different stages in the lifecycle of Storage Foundation files.
+
+#### Opening, writing, reading, closing
+
+```js
+// Open a file (creating it if needed).
+const file = await storageFoundation.open('test_file');
+try {
+  // Request 100 bytes of capacity for this context.
+  await storageFoundation.requestCapacity(100);
+
+  const writeBuffer = new Uint8Array([64, 65, 66]);
+  // Write the buffer at offset 0. After this operation, `result.buffer`
+  // contains the transferred buffer and `result.writtenBytes` is 3,
+  // the number of bytes written. `writeBuffer` is left detached.
+  let result = await file.write(writeBuffer, 0);
+
+  const readBuffer = new Uint8Array(3);
+  // Read at offset 1. `result.buffer` contains the transferred buffer,
+  // `result.readBytes` is 2, the number of bytes read. `readBuffer` is left
+  // detached.
+  result = await file.read(readBuffer, 1);
+  // `Uint8Array(3) [65, 66, 0]`
+  console.log(result.buffer);
+} finally {
+  file.close();
+}
+```
+
+#### Opening, listing, deleting
+
+```js
+// Open three different files (creating them if needed).
+await storageFoundation.open('sunrise');
+await storageFoundation.open('noon');
+await storageFoundation.open('sunset');
+// List all existing files.
+// `["sunset", "sunrise", "noon"]`
+await storageFoundation.getAll();
+// Delete one of the three files.
+await storageFoundation.delete('noon');
```

Comment:
@fivedots This example from the explainer actually doesn't work. It fails because the file is open. See https://github.com/WICG/storage-foundation-api-explainer/issues/24 for background.